src/emit/substrate/mod.rs

```diff
@@ -22,6 +22,109 @@ pub(super) mod target;
 // When using the seal api, we use our own scratch buffer.
 const SCRATCH_SIZE: u32 = 32 * 1024;
 
+#[macro_export]
+macro_rules! emit_context {
+    ($binary:expr) => {
+        #[allow(unused_macros)]
+        macro_rules! byte_ptr {
+            () => {
+                $binary.context.i8_type().ptr_type(AddressSpace::Generic)
+            };
+        }
+
+        #[allow(unused_macros)]
+        macro_rules! cnst {
```

Comment:
`cnst` is very short. How about spelling out `const` with the vowel.

Reply:
ACK